REPs/REP-26.md

```diff
@@ -37,3 +38,8 @@ The current staking mechanism, which mints a Chip for each unit of \$RSS3 staked
 ## Rationale
 
 The core of this proposal is to reduce the number of Chips minted to one for each staking transaction, regardless of the amount staked, where each Chip will represent the total value staked.
+
+## Reference Implementations
+
+1. Whitepaper: [RSS3-Network/Whitepaper#7](https://github.com/RSS3-Network/Whitepaper/pull/7) and [RSS3-Network/Whitepaper#8](https://github.com/RSS3-Network/Whitepaper/pull/8)
```

Comment:
change to list:
1. Whitepaper
    1. [RSS3-Network/Whitepaper#7](https://github.com/RSS3-Network/Whitepaper/pull/7). 
    1.  [RSS3-Network/Whitepaper#8](https://github.com/RSS3-Network/Whitepaper/pull/8)